faqs/galaxy/dataset_tool_erorrs_metadata.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Tool Errors related to Metadata
+area: datasets   
+description:  Finding and Correcting Metadata
+contributors: jennaj,
```

Comment:
It looks weird here.

Reply:
My mistake.

Reply:
it should not be removed but updated to reflect the effective list of current contributors. I guess you want to be in that list too.

Reply:
Yes, I'm aware, I was confused as to whose names to be added? Just mine or also someone else's?

Reply:
there's not such thing as mandatory contributors here :) Here's the explanation: jennaj wrote the FAQ initially, so she has to be on the contributors list. You moved it to the GTN, so you also deserve to be there. Just add both names (please check how it's done in the other PRs)